extensions/tensorflow/TFApplyGraph.cpp

```diff
@@ -221,7 +224,7 @@ int64_t TFApplyGraph::TensorWriteCallback::process(const std::shared_ptr<io::Bas
   auto num_wrote = stream->write(reinterpret_cast<uint8_t *>(&tensor_proto_buf[0]),
                                      static_cast<int>(tensor_proto_buf.size()));
 
-  if (num_wrote != tensor_proto_buf.size()) {
+  if (static_cast<uint64_t>(num_wrote) != tensor_proto_buf.size()) {
```

Comment:
same here: `stream->size()` returns `size_t`, so we should preferably cast `num_wrote` to that.

Reply:
Updated in 7f2abdc0ae7a9f55c07339edc9c9d84de409c4de